packages/skygear-core/src/core-js.ts

```diff
@@ -1,3 +1,7 @@
 declare module "core-js-pure/features/url" {
   export default URL;
 }
+
+declare module "core-js-pure/features/url-search-params" {
+  export default URLSearchParams;
+}
```

Comment:
Sorry, this file should be called `core-js.d.ts` to signify it is a declaration file. I should have told you this yesterday. My bad.

Reply:
Updated in 91f4bb6. I updated the `clean-decl` command too, see if it looks ok?